### PR TITLE
Setting the Client_Key of MicroXRCE-Agent

### PR DESCRIPTION
**Pull request by taro83**
Added an argument in Node() that can set Client Key of Micro XRCE-Agent at node initialization

### CODE REVIEW

**Comment by OpusK** [comment]
Hi, @taro83 

Thank you for your contribution.
I'm just leaving a few reviews, can you fix it?

**Comment by OpusK** [comment]
Please delete the files below and add them to .gitignore if necessary.
- .DS_Store 
- src/.DS_Store

File: src/ros2/ros2.hpp [hunk 1]
```diff
@@ -24,10 +24,10 @@ void runNodeSubUserCallback(uint16_t id, void* msgs, void* args);
 class Node
 {
   public:
-    Node(const char* node_name = "ros2_xrcedds_participant");
+    Node(const char* node_name = "ros2_xrcedds_participant",unsigned int client_key);
```
Comment (OpusK) [inline src/ros2/ros2.hpp]: Set the default value for the client_key parameter for both functions 27 and 30 as shown below.

- Node(const char* node_name = "ros2_xrcedds_participant", unsigned int client_key = 0xAABBCCDD);
- void recreate(const char* node_name = "ros2_xrcedds_participant", unsigned int client_key = 0xAABBCCDD);

**Comment by taro83** [comment]
Hi @OpusK 
Thank you for your comment.
I added setting to ignore .DS_Store.

I will try to fix any other suggestions from you.

**Comment by OpusK** [comment]
@taro83 

You added .gitignore but didn't delete the DS_Store files.
Please delete it.

**Comment by taro83** [comment]
@OpusK 
Sorry, my mistake.
I deleted DS_Store files.

**Comment by OpusK** [comment]
@taro83 

Thanks for your contribution.
We will release this soon.

**Comment by taro83** [comment]
Thank you for merge ! 
I will follow this good project.

**Pull request merged**